scripts/rebuild-filterlist.mjs

```diff
@@ -28,4 +193,18 @@ ${fs.readFileSync(path.join(rulesDir, 'filterlists', 'easylist_cookie_allowlist_
 ${fs.readFileSync(path.join(rulesDir, 'filterlists', 'overrides.txt'), 'utf-8')}
 `;
 
-fs.writeFile(path.join(rulesDir, 'filterlist.txt'), filterlistContent, () => console.log('Written filterlist.txt'));
+    fs.writeFile(path.join(rulesDir, 'filterlist.txt'), filterlistContent, () => console.log('Written filterlist.txt'));
+}
+
+/**
+ * Parse and refine lists using domain popularity, then combine them and write filterlist.txt
+ */
+async function rebuildFilterList() {
+    await loadTrancoList();
+    await processFilterList('easylist_cookie_specific_uBO.txt');
```

Comment:
should we call processFilterlist() on all input files, not just "specific" ones?